### DISCUSSION TITLE
What's New

### DISCUSSION THREAD

**Discussion by rcalixte** [linuxmint/discussions#39] [Tutorials]
While researching another issue, it was discovered that it seems that some new features in Cinnamon are not being communicated in a way that the typical user is receiving them. [Example YouTube video](https://youtu.be/uVWzsDDgUDM?t=485) (starting at the relevant mark, about 40 seconds in duration)

Would it make sense to bundle a "What's New" application window that would just be the cumulative `debian/changelog` file for the current major release (e.g. all 5.8.x updates rolled up until 6.0.x)? Would it make sense to commission an official Cinnamon (not just Linux Mint) YouTube channel specifically to highlight changes or lesser-known features?

**Comment by JosephMcc** [comment]
For Mint, I'm pretty sure there is always a "What's new" that details these things on the blog for a new release. I'd have to double check but I'm pretty sure the upgrade tool also contains a link to this.

**Reply by rcalixte** [reply]
Correct, I should make clear for the non-Mint users of Cinnamon and for those that do not consume the blog. Also, it is worth noting that the example from the video above was not called out in any blog posts.

**Reply by rcalixte** [reply]
[New Features in Linux Mint 21.2 (Cinnamon 5.8)](https://www.linuxmint.com/rel_victoria_cinnamon_whatsnew.php)

**Comment by anaximeno** [comment]
There should be a section referencing Mint's Blog in the welcome application at least.
